action.yml

```
@@ -13,6 +13,9 @@ inputs:
   versionApiURL:
     description: "Trybe Application Version env URL to access"
     required: true
+  herokuHealthApiURL:
```

Estar no heroku ou outro lugar é irrelevante, então da pra remover do nome do atributo.

```suggestion
  applicationHealthApiURL:
```